deployment/sriov-network-operator-chart/templates/pre-delete-webooks.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
```

Comment:
nit: WDYT about adding some comments about what this job does?

Reply:
Sure, I will add more details